Documentation/Metrics/arangodb_replication2_replicated_state_follower_number.yaml

```diff
@@ -0,0 +1,12 @@
+name: arangodb_replication2_replicated_state_follower_number
+introducedIn: "3.10.0"
+help: |
+  Number of follower replicated states.
+unit: number
+type: counter
+category: Replication
+complexity: simple
+exposedBy:
+  - dbserver
+description: |
+  Number of replicated states this server is a leader of.
```

Comment:
Looks like you switched leader and follower in the updated descriptions :)

Reply:
Thanks for spotting!